OracleOGG/MySQL/12.2.0.1.1/Dockerfile

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Since: June, 2017
+# Author: hadi.koesnodihardjo@oracle.com
+# Description: script to build a Docker image for Oracle GoldenGate for non-Oracle (MySQL).
+#
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
+#
+#
+FROM mysql/mysql-server:5.7.18
```

Comment:
This is a very specific MySQL Server version. Are you signing up to update this every time they release a minor version in the 5.7 family?